sdk/storage/storage-blob/src/ContainerClient.ts

```diff
@@ -1403,18 +1396,6 @@ export class ContainerClient extends StorageClient {
         ...convertTracingToRequestOptionsBase(updatedOptions),
       });
 
-      response.segment.blobItems = [];
```

Comment:
if you delete this logic, remember to update the min version of core-http or you will have a nasty surprise :)

Reply:
done